### DISCUSSION TITLE
Why Multiple Installation Methods?

### DISCUSSION THREAD

**Discussion by johnsoga** [Homebrew/discussions#5072] [Getting started]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

It appears there are now 2 ways to install Homebrew. The original bash script installation method and now an alternative .pkg installer option. Is there a reason for the second installer option now? I know there have been complaints about the security posture of the bash script option, which personally I've never had issue with. Not to mention that it tells you exactly what is doing. The package installer option I feels add a layer of abstraction from the user to know what exactly is happening behind the scenes. There is definitely controversy around MacOS installers and privileges they have to install malicious code simply upon being opened let alone again the user having no real idea what is happening behind the scenes. Are the two installer options truly a 1-1 match in terms of what they are doing and if so, then again, why have both options? This is a question of curiosity and not meant to be seen as a negative critique of the choice.

**Comment by SMillerDev** [comment]
You can't install a pkg on Linux and I made the pkg installer because it's hell to run the bash script for a fleet of devices.
